src/main/java/net/sf/jabref/event/AddEntryEvent.java

```diff
@@ -0,0 +1,24 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+/**
+ * Instances of this class are suitable for passing through the event bus.
+ * A <code>BibEntry</code> object the changes were applied on is required.
+ * <code>AddEntryEvent</code> should be used, when new <code>BibEntry</code>
+ * was added to the database.
+ */
+
+public class AddEntryEvent {
+
+    private final BibEntry bibEntry;
+
+    public AddEntryEvent(BibEntry bibEntry) {
```

Comment:
Yeah, this is the really right implementation. I discussed away the old inheritance, because there was Object casting, but this proposal sounds great.